### DISCUSSION TITLE
Slow nova-api calls to actions/cards/lenses/filters.

### DISCUSSION THREAD

**Discussion by AEmad01** [laravel/nova-issues#3306] [Q&A] [answered]
- Laravel Version: 8.12
- Nova Version: 3.20.0
- PHP Version: 7.4.14
- Database Driver & Version: MYSQL 10.4.17-MariaDB
- Operating System and Version: Windows 10
- Browser type and version: Google Chrome 89.0.4389.90
- Reproduction Repository: -

### Description:
nova-api calls to actions/lenses/filters/cards take at least a second to complete, regardless of the amount of data. 
![wat](https://user-images.githubusercontent.com/47536525/112282727-08750c80-8c90-11eb-960b-b1ac8a3511cc.png)

Running on localhost with Apache/2.4.46, also tried running Nginx.

![EYisuhF](https://user-images.githubusercontent.com/47536525/112282139-7cfb7b80-8c8f-11eb-82bc-9f1305ca5e4f.png)

**Comment by crynobone** [comment]
Majority of the waiting for the Laravel application to boot up and handle the request. This is where Laravel Octane will be beneficial to reduce the requirement to boot the framework on each request.

Feel free to continue the discussion, or point out if there anything specific that we can optimize, as this request being slow has been reviewed few times in the past.

#442 #3127

**Comment by AEmad01** [comment, marked as answer]
Brought the request time to 100ms by enabling opcache and using php 8.
